locales/en-US/formautofill.properties

```diff
@@ -3,26 +3,36 @@
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 preferenceGroupTitle = Form Autofill
-enableProfileAutofill = Enable Profile Autofill
-savedProfiles = Saved Profiles…
+enableAddressAutofill = Autofill addresses
+learnMore = Learn more
+savedAddresses = Saved Addresses…
+enableCreditCardAutofill = Autofill credit cards
```

Comment:
Same here, context. What's this? A checkbox in preferences, a title, a button? Please add a localization comment.

Reply:
I think I'll make it `autofillCreditCardsCheckbox`.